src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -91,7 +91,27 @@ public IOperation GetOperation(SyntaxNode node, CancellationToken cancellationTo
         /// </summary>
         public ImmutableArray<Operations.BasicBlock> GetControlFlowGraph(Operations.IBlockOperation body)
```

Comment:
>GetControlFlowGraph [](start = 53, length = 19)

We should make this method static. I wasn't sure if the feature flag implementation would depend of SemanticModel instance and left it an instance method.

Reply:
I agree. @AlekseyTs can you please make this change in one of your PRs?